src/commands/deploy.ts

```diff
@@ -57,15 +58,15 @@ export async function deployCommand(options: any) {
   }
 
   // check if user is logged in
-  if (configuration.cloudProvider !== "selfHostedAws") {
+  if (configuration.cloudProvider !== CloudProviderIdentifier.SELF_HOSTED_AWS) {
```

Comment:
What do you think to add here a deprecation warning for the user if he uses `aws`?